File: Python/Processor.py
class Processor():

    #private attributes
    name = ""
    price = 0

    #constructor
    def __init__(self, name = "", price = 0):
        self.name = name
        self.price = price

    def setName(self, n):
        self.name = n

    def getName(self):
        return self.name

    def setPrice(self, x):
        self.price = x

    def getPrice(self):
        return self.price

File: PHP/Processor.php
<?php

class Processor
{
    private $name = "";
    private $price = 0;

    public function __construct($name = "", $price = 0)
    {
        $this->name = $name;
        $this->price = $price;
    }

    public function setName($n = "")
    {
        $this->name = $n;
    }
    public function getName()
    {
        return $this->name;
    }

    public function setPrice($x = 0)
    {
        $this->price = $x;
    }
    public function getPrice()
    {
        return $this->price;
    }

    public function __destruct(){}
}

?>